src/EA.Iws.Web/Infrastructure/xlsxActionResult.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace EA.Iws.Web.Infrastructure
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.IO;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using System.Web;
+    using System.Web.Mvc;
+    using ClosedXML.Excel;
+
+    public class XlsxActionResult<T> : FileResult
+    {
+        private readonly IEnumerable<T> data;
+        private const string contentType = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet";
```

Comment:
In EA.Iws.Web.Infrastructure there is a MimeTypes static class with an MSExcelXml const. You can use that instead of needing the const in here.